logging/OpenShift.md

```diff
@@ -50,18 +50,27 @@ oc login [cluster-hostname] -u [userID]
 
 Customization of SAS Viya Logging on OpenShift deployment follows the same process as in a standard logging deployment, which uses the USER_DIR environment variable to specify the location of your customization files. See the [logging README](../logging/README.md#log_custom) for information about the customization process.
 
-### OpenShift-Specific Customization
+### OpenShift-Specific Customization and Information
 
 #### OpenShift Infrastructure Message Retention
-You can change the number of days that log messages from OpenShift infrastructure namespaces (any namespace that starts with "openshift") 
-are retained so that the messages do not fill up the storage space. To change the retention period, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable. The default value is `1` (1 day).
+An additional index management policy, named `viya_infra_idxmgmt_policy`, is added 
+in an OpenShift environment. The policy manages log messages from OpenShift 
+infrastructure namespaces (which is ny namespace that starts with "openshift"). 
+Because the OpenShift infrastructure namespaces can generate a large number of 
+log messages, you can use the policy to change the number of days that these 
+log messages are retained. To change the retention period, modify the `INFRA_LOG_RETENTION_PERIOD` environment variable. The default value is `1` (1 day).
+
+***Note:*** See [Log Retention](Log_Retention.md) for information 
+about the index policies and log message retention for SAS Viya, Kubernetes, 
+and logging components.
 
 #### Access Using Route Objects
 OpenShift uses [route](https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html) objects, a feature unique to OpenShift, to access Kibana and (optionally) the Elasticsearch API endpoint. This makes it unnecessary to configure ingress objects or surface nodePorts.
 No customizations are required, even if you are using ingress, because the `deploy_logging_open_openshift.sh` script defines a route for Kibana.
 
 #### TLS
-OpenShift route objects are always TLS-enabled. Traffic is encrypted between the user 
+OpenShift route objects are TLS-enabled. The routes are configured 
+with re-encrypt termination, which means that traffic is encrypted between the user 
 and OpenShift and again within the OpenShift cluster. The TLS certificates used on 
 OpenShift come from the same Kubernetes secrets as a standard logging deployment:
 ***In-Cluster TLS:***
```

Comment:
Please double-check the placement of the **In-Cluster TLS** and Ingress **TLS headers**.  It looks fine here but when I opened the file directly, it appears the line-breaks before each of those headers had been lost (the headers appear to be part of the preceding line.

Reply:
Spacing fixed. They weren't coded as headers, but just highlighted text. Even though I had them on separate lines, it messed up the placement until I put a blank line before each one.